git_changes.sh

```diff
@@ -11,10 +25,42 @@ current_branch=$(git rev-parse --abbrev-ref HEAD)
 git pull origin "$current_branch"
 
 new_head=$(git log | head -1 | awk '{print $2}')
-if [[ "$current_head" == "$new_head" ]]; then
+if [ "$current_head" == "$new_head" -a "$files_count" -gt 0 ]; then
```

Comment:
The `files_count` variable being 0 here isn't really indicative of there not being any new commits, right? That seems like a different issue. If so, perhaps that should be its own conditional/echo+exit block?

Reply:
@lsitu - Ah I see, it's the initial run you're concerned about since current and new head values would be the same initially. No I think this works :+1: thanks for clarifying